### ISSUE TITLE
As a user, I want to automatically receive adopted assets (where !mad) 

### ISSUE THREAD

**Issue by rhlsthrm** [connext/nxtp#1057]
**Background**
In order to bridge cross-chain where local mad asset != adopted, we must automatically swap into the adopted assets

**Suggested implementation**
On execute, Amarok stableswaps automatically swap from local<>adopted

**Acceptance criteria**
1. User automatically received adopted assets
 
 **Out of scope**
 Assume no slippage limits (this will be handled #1358

**Subtasks**
- [ ]

**Comment by rhlsthrm** [comment]
Blocked by UI.

**Comment by alexwhte** [comment]
Blocked by https://github.com/connext/nxtp/issues/1358

**Comment by alexwhte** [comment]
Closing per hack
